loan-payment-skip/LETTERSPECS/BANNO.LOANPAYMENT.SKIP.CFG

```diff
@@ -159,3 +274,5 @@ TC:A single blank line (as above) forces a new line in the display.
 TC:
 TC:
 TC:Two blank lines in a row (as above) forces a new paragraph.
+TC:<b>HTML Ready?</b>

```

Comment:
In the Config file line 277, do you know yet if the HTML tags are working? If so I suggest adding the useable HTML tags to the comments section of Terms and Conditions.

Reply:
HTML tags not working yet - s/b shortly but not sure we want to hold this up waiting for them.  Removing the HTML reference.